### PR TITLE
Fix for issue 3541

### PR DESCRIPTION
**Pull request by srampal**

### CODE REVIEW

**Comment by srampal** [comment]
@sdodson @jcpowermac @danyc97  PTAL

File: roles/contiv/tasks/download_bins.yml [hunk 1]
```diff
@@ -25,3 +25,20 @@
...
+    mode: 0755
+    validate_certs: False
+  environment:
+    http_proxy: "{{ http_proxy|default('') }}"
+    https_proxy: "{{ https_proxy|default('') }}"
+    no_proxy: "{{ no_proxy|default('') }}"
+
+- name: Download Bins | Extract cni tar file
+  unarchive:
+    src: "{{ cni_download_dir }}/cni-{{ cni_version }}.tbz2"
```
Comment (jcpowermac) [inline roles/contiv/tasks/download_bins.yml]: @srampal the `src` of the `unarchive` action could be a variable that was registered from `get_url`, e.g.

```
- name: Download Bins | Download cni tar file
  get_url:
    url: "{{ cni_bin_url }}"
    dest: "{{ cni_download_dir }}"
    mode: 0755
    validate_certs: False
  environment:
    http_proxy: "{{ http_proxy|default('') }}"
    https_proxy: "{{ https_proxy|default('') }}"
    no_proxy: "{{ no_proxy|default('') }}"
  register: download_file

- name: Download Bins | Extract cni tar file
  unarchive:
    src: "{{ download_file.dest }}"
    dest: "{{ cni_download_dir }}"
    copy: no
  when: download_file.changed
```
Also adding a when clause is a further step to idempotent playbook.

Comment (srampal) [inline roles/contiv/tasks/download_bins.yml]: @jcpowermac  ok .. made the change and resubmitted the PR ... PTAL

**Comment by DanyC97** [comment]
@srampal looks alright to me, cheers

**Comment by sdodson** [comment]
[merge]

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
flake https://github.com/openshift/origin/issues/13271
[merge]

**Comment by sdodson** [comment]
> If flaky tests caused the job to fail, leave a comment with links to the GitHub issue(s) in the openshift/openshift-ansible repository with the kind/test-flake label that are tracking the flakes. If no issue already exists for the flake you encountered, create one.

@stevekuznetsov i can't link origin flakes?

**Comment by stevekuznetsov** [comment]
@sdodson I can change it to be origin flakes, but it's one repo and one repo only ATM

**Pull request merged**